Apps/Examples/Examples/All Examples/Custom2DPuckExample.swift

```diff
@@ -17,11 +19,72 @@ public class Custom2DPuckExample: UIViewController, ExampleProtocol {
         let configuration = Puck2DConfiguration(topImage: UIImage(named: "star"))
         mapView.location.options.puckType = .puck2D(configuration)
         mapView.location.options.puckBearingSource = .course
+
+        // Center map over the user's current location
+        mapView.mapboxMap.onNext(.mapLoaded, handler: { [weak self] _ in
+            guard let self = self else { return }
+
+            if let currentLocation = self.mapView.location.latestLocation {
+                let cameraOptions = CameraOptions(center: currentLocation.coordinate, zoom: 17.0)
+                self.mapView.camera.ease(to: cameraOptions, duration: 2.0)
+            }
+        })
+
+        // Setup a toggle button to show how to toggle the accuracy radius
+        mapView.mapboxMap.onEvery(.cameraChanged, handler: { [weak self] _ in
+            guard let self = self else { return }
+
+            if self.mapView.cameraState.zoom >= 17.0 {
+                if let button = self.toggleAccuracyRadiusButton {
+                    button.isHidden = false
+                } else {
+                    self.setupToggleShowAccuracyButton()
```

Comment:
I think it'd simplify the code if we set up the toggle button unconditionally at the beginning of view did load and then assume it exists in this camera changed handler.

Reply:
After you make that change, I'd like to review at this handler again